client/package.json

```diff
@@ -9,11 +9,15 @@
     "web": "expo start --web"
```

Comment:
In `app.json`:
Change `"orientation": "portrait"` to `"orientation": "default"`.
Now you can check how the app looks in landscape display whenever you flip your device.